workers/Dockerfile

```diff
@@ -0,0 +1,41 @@
+# Build generic worker and run
+
+FROM golang:1.18-buster as build
```

Comment:
Could do this in a future PR, but I think this go version should be updated by our generators.